### PR TITLE
filetree: Update example and documentation for plugin

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Fixes: #727

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
changelogs/fragments/727_filetree_example.yml
plugins/lookup/dig.py
plugins/lookup/filetree.py

### CODE REVIEW

**Comment by jpmens** [comment]
shipit

File: changelogs/fragments/727_filetree_example.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
```
Comment (felixfontein) [inline changelogs/fragments/727_filetree_example.yml:1]: I don't think this needs a changelog fragment, as it's a docs only change (and not fixing something that was terribly wrong).

File: plugins/lookup/filetree.py [hunk 2]
```diff
@@ -45,46 +45,50 @@
     mode: '{{ item.mode }}'
   with_filetree: web/
   when: item.state == 'link'
+
+- name: list all files under web/
+  ansible.builtin.debug:
+    msg: "{{ lookup('filetree', 'web/') }}"
```
Comment (felixfontein) [inline plugins/lookup/filetree.py:51]:
```suggestion
    msg: "{{ lookup('community.general.filetree', 'web/') }}"
```
(The others are changed in #722.)

**Comment by Akasurde** [comment]
@jpmens @felixfontein Thanks for the review.

**Comment by felixfontein** [comment]
@Akasurde thanks for fixing this!

**Pull request merged**